### PR TITLE
Add test for menu_anchor.2.dart

### PR DESCRIPTION
**Pull request by ValentinVignal**
Contributes to https://github.com/flutter/flutter/issues/130459

It adds a test for
- `examples/api/lib/material/menu_anchor/menu_anchor.2.dart`

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] I followed the [breaking change policy] and added [Data Driven Fixes] where supported.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[Data Driven Fixes]: https://github.com/flutter/flutter/wiki/Data-driven-Fixes

### CODE REVIEW

File: examples/api/test/material/menu_anchor/menu_anchor.2_test.dart [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    );
+
+    await tester.tap(find.byIcon(Icons.more_horiz));
+    await tester.pumpAndSettle();
+
+    for (int i = 1; i <= 3; i++) {
+      expect(find.widgetWithText(MenuItemButton, 'Item $i'), findsOne);
+    }
+  });
+
```
Comment (bleroux) [inline examples/api/test/material/menu_anchor/menu_anchor.2_test.dart:28]: Consider removing this empty line

**Review by bleroux — APPROVED** [review]
LGTM with one small nit.
Thanks for the contribution!

File: examples/api/test/material/menu_anchor/menu_anchor.2_test.dart (hunk 1 above)
Comment (ValentinVignal) [reply examples/api/test/material/menu_anchor/menu_anchor.2_test.dart]: Argh!! Always this empty line 😱 😞 

I removed it in
[test: Remove empty line](https://github.com/flutter/flutter/pull/151399/commits/7fa738679b0ca0c8754291a3b1e87e9804265247)

File: examples/api/test/material/menu_anchor/menu_anchor.2_test.dart [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/menu_anchor/menu_anchor.2.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('The menu should display 3 items', (WidgetTester tester) async {
```
Comment (TahaTesser) [inline examples/api/test/material/menu_anchor/menu_anchor.2_test.dart:10]:
```suggestion
  testWidgets('The menu should display three items', (WidgetTester tester) async {
```

**Review by TahaTesser — APPROVED** [review]
LGTM with nit

**Pull request merged**